test/local/token_expiry_tests.js

```diff
@@ -6,7 +6,7 @@ var test = require('../ptaptest')
 var TestServer = require('../test_server')
 var Client = require('../client')
 
-process.env.PASSWORD_CHANGE_TOKEN_TTL = '-1'
+process.env.PASSWORD_CHANGE_TOKEN_TTL = '1'
```

Comment:
FWIW, I tried using `0` for duration here and got failing tests. I originally thought that this was convict not handling the duration correctly, but [it does](https://github.com/mozilla/node-convict/blob/master/lib/convict.js#L50).